.travis.yml

```diff
@@ -6,8 +6,6 @@ addons:
     - python-numpy
     - python-matplotlib
     - python-scipy
-    - python-pandas
-    - python-seaborn
```

Comment:
seaborn is indeed not in the standard Ubuntu (actually it's in the 16.04), but I am surprised that we can't get pandas.

I think that the strategy would be pandas via ubuntu. It has compiled code, so it's easier that way, and seaborn via pip.